common/src/JetCorrections.cxx

```diff
@@ -268,31 +453,31 @@ std::unique_ptr<FactorizedJetCorrector> build_corrector(const std::vector<std::s
   //propagate to MET
   //apply type1 MET correction to RAW MET
   //NB: jet with substracted muon Pt should be used
-
-  void correct_MET(const Event & event, const bool & do_L1corr, const bool do_L1RCcorr, FactorizedJetCorrector & corrector_L1RC){
+ void correct_MET(const Event & event, const bool & do_L1corr, const bool do_L1RCcorr, FactorizedJetCorrector & corrector_L1RC, FactorizedJetCorrector & corrector){
```

Comment:
Does this mean you don't need the `corrector` arg anymore?

Reply:
Yes, thats true. Totally missed this point. Thanks.